Microsoft.Toolkit.Win32/Microsoft.Toolkit.Wpf.UI.Controls/InkToolbar/InkToolbar.cs

```diff
@@ -28,7 +28,6 @@ public class InkToolbar : WindowsXamlHostBaseExt
         public InkToolbar()
             : this(typeof(Windows.UI.Xaml.Controls.InkToolbar).FullName)
         {
-            this.Visibility = System.Windows.Visibility.Collapsed;
```

Comment:
Don't know exactly why, but removing that like broke this control for me. It just doesn't show up in the UI at all.

Reply:
I forgot why that was there.  That's a workaround for a bug in the underlying control.

Reply:
@GraniteStateHacker @azchohfi How are we going to handle continuous testing of this? I doubt we want to expend energy to test on every SDK and OS build released manually.

Reply:
It feels like tests for issues like this need to be incorporated in the OS's or SDK's smoke tests.

Reply:
A much better workaround is in. (and commented)